css/style.css

```diff
@@ -31,13 +31,13 @@ a {
 }
 
 .main {
-  background: #fff url(../img/header-shapes-mobile.svg) no-repeat;
+  background: #fff url(../img/bg-header-desktop.svg) no-repeat;
```

Comment:
- Try to implement your:
@media screen and (min-width: 1080px) { // please note that it doesn’t have to be exactly 1080px, any value between 992 and 1100 is ok.
/*
.
.
*/
}

- Fix your element positions: 
![Firefox_Screenshot_2021-08-20T02-07-52 188Z](https://user-images.githubusercontent.com/22258274/130167972-6c90a1c7-0a26-4efe-9f63-57a2d1ead340.png)

Reply:
I'm very confused about what you meant here. I can't reproduce what is showing in your photo.

Reply:
- [ ]  Kindly check your project with inspect tool and up `796` size and there you will see your `projects cards` are overflowing to the right side and your `header and intro` sections are not. I think you should fix your `projects` overflow. The first reviewer wanted to explain this problem to you with the image. Small hint try to use `max-width` to your container it will give more flexibility. I hope you get the point and other things are excellent. Good job 👏. Please if there is any problem send me message via Slack.